flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/iceberg/TestIcebergPropertiesConverter.java

```diff
@@ -50,7 +52,11 @@ void testCatalogPropertiesWithHiveBackend() {
             IcebergPropertiesConstants.ICEBERG_CATALOG_URI,
             "hive-uri",
             IcebergPropertiesConstants.ICEBERG_CATALOG_WAREHOUSE,
-            "hive-warehouse"),
+            "hive-warehouse",
```

Comment:
key1 and value1 should not exists in the result.  By design only the keys defined by Graviitno like `catalog-backend` or start with `flink.bypass.` are passed to the underlying flink connector. the other property keys are dropped

Reply:
The current unified logic is to automatically add a prefix to parameters that have not been mapped and have not been added with "Flink. bypass." before passing them to Gravitino. When setting them, there is no mandatory restriction that they must include "Flink. bypass, This is also for compatibility with old logic, but what you said is also reasonable. Let me unify it first

Reply:
Got it, the origin properties transform logic in Hive maybe bugy, cc @coolderli

Reply:
@sunxiaojian ,  how about following the logics here?

When creating a catalog, the properties managed by Gravitino are transformed, the properties starting with `flink.bypass` are left, and the other properties will add the `flink.bypass` prefix.

When loading a catalog,  the properties managed by Gravitino are transformed, the properties starting with `flink.bypass` are removed from the prefix, and others are dropped.

Reply:
revert

Reply:
@FANNG1 Ohh, this is the logic before this modification.